src/Traits/HasPermissions.php

```diff
@@ -113,10 +114,21 @@ public function hasPermissionTo($permission, $guardName = null): bool
         $permissionClass = $this->getPermissionClass();
 
         if (is_string($permission)) {
-            $permission = $permissionClass->findByName(
-                $permission,
-                $guardName ?? $this->getDefaultGuardName()
-            );
+            try {
+                $permission = $permissionClass->findByName(
+                    $permission,
+                    $guardName ?? $this->getDefaultGuardName()
+                );
+            } catch (PermissionDoesNotExist $e) {
+                // the permission as string is not present in the database
+                // so check wildcard permission if enabled in config
+                // Use sensitive default for backwards compatibility
```

Comment:
Perhaps reword for brevity and clarity? Something like:
```
// Arriving here means the permission is not found
// so check wildcard permission if enabled.
```

Reply:
pls see my new comment